pkg/restapi/operation/operations_test.go

```diff
@@ -190,6 +192,20 @@ func (m *mockEDVStore) Query(*models.Query) ([]string, error) {
 	return []string{"docID1", "docID2"}, nil
 }
 
+type mockUUIDBuilder struct {
+	testUUIDString string
+}
+
+// mock uuid().New() to replicate generating duplicated uuid vault names
```

Comment:
I think there should be some tweaks to how this comment is written...

That New() method itself really doesn't have anything to do with vaults, so "vault names" needn't appear in the comment... (I believe this is an example of leaking of abstractions)

However, it makes sense to explain the purpose of this above the `mockUUIDBuilder` type.

Also, there are a few minor grammatical errors in the comment...

Should be: `mocks` instead of `mock` and `uuid` should be capitalized (the second one in the commend, not the method name).

Reply:
fixed